src/internal/orders/sales-order-identifier.ts

```diff
@@ -4,7 +4,7 @@ import { hideAndFreeze, Identifiers, Joi, _internal } from "../common";
 
 export abstract class SalesOrderIdentifierBase implements ISalesOrderIdentifier {
   public readonly id: string;
-  public readonly identifiers: Identifiers;
+  public readonly identifiers?: Identifiers;
```

Comment:
@justin-shipstation make this required and just let it initialize to an empty object